app/controllers/projects/transfer_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Projects
+  # Controller for transferring projects to another namespace
+  class TransferController < Projects::ApplicationController
+    before_action :project, only: %i[create]
+    before_action :authorized_namespaces, only: %i[create]
+
+    def create
+      if Projects::TransferService.new(@project, current_user).execute(new_namespace)
+        flash[:success] = t('.success', project_name: @project.name)
+        respond_to do |format|
+          format.turbo_stream { redirect_to project_path(@project) }
+        end
+      else
+        @error = @project.errors.messages.values.flatten.first
+        render :edit, status: :unprocessable_entity
+      end
+    end
+
+    private
+
+    def project_params
```

Comment:
**nitpick (blocking):** Please remove this, this looks to be unintentionally copied over.